### PR TITLE
Rpm lint checks

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
We probably could also delete the C header files, but I am reluctant mess around with the gem sources.

Let me know if you have a better solution for this.

**Comment by ChrisBr** [comment]
> Many of the bundled gems we package contain C header files.
Since we won't be able to solve all these cases upstream in
reasonable time we disable that check for now.

[nitpick] Don't like this description too much because this is intended by upstream, there is nothing we could fix about it so it probably will stay like this 'forever'.

**Comment by bgeuken** [comment]
Fair enough, I change it:+1:

File: dist/obs-rpmlintrc [hunk 1]
```diff
@@ -8,3 +8,4 @@ addFilter('zero-length')
 addFilter('files-duplicate')
 addFilter('script-without-shebang')
 addFilter('devel-file-in-non-devel-package')
+addFilter('unexpanded-macro')
```
Comment (ChrisBr) [inline dist/obs-rpmlintrc]: The commit message is wrong, it says ``Rpmlint: Silence devel-file-in-non-devel-package warning`` but you actually add the ``unexpanded-macro`` rpmlintrc fiter.

**Pull request merged**